tools/table_annovar/table_annovar.xml

```diff
@@ -63,21 +63,21 @@
         <validator type="unspecified_build" />
     </param>
 
-    <param name="gene_anns" type="select" multiple="True" optional="True" label="Gene Annotations" help="" >
+    <param name="gene_anns" type="select" display="radio" multiple="True" optional="True" label="Gene Annotations" help="" >
```

Comment:
I think `display="checkboxes"` would be better. Radio buttons are normally used for mutually exclusive options, while here multiple selections are explicitly allowed: https://www.nngroup.com/articles/checkboxes-vs-radio-buttons/